README.md

```diff
@@ -248,6 +248,32 @@ Bats. For example, to install Bats into `/usr/local`,
 Note that you may need to run `install.sh` with `sudo` if you do not
 have permission to write to the installation prefix.
 
+## Running Bats in Docker
+
+Check out a copy of the Bats repository, then build a container image:
+
+    $ git clone https://github.com/bats-core/bats-core.git
+    $ cd bats-core
+    $ docker build --tag bats:latest .
+
+This creates a local Docker image called `bats:latest` based on [Alpine
+Linux](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md).
+To run Bats' internal test suite (which is in the container image at
+`/opt/bats/test`):
+
+    $ docker run -it bats:latest /opt/bats/test
+
+To run a test suite from your local machine, mount in a volume and direct
+Bats to its path inside the container:
+
+    $ docker run -it -v "$(pwd):/code" bats:latest /code/test
+
+This is a minimal image. If more tools are required this can be used as a
+base image in a Dockerfile using `FROM <Docker image>`.
+In the future there may be images based on Debian, and/or with more tools
+installed (`curl` and `openssl`, for example). If you require a specific
+configuration please search and +1 an issue or
+[raise a new issue](https://github.com/bats-core/bats-core/issues).
```

Comment:
Definitely, I think this will develop in a similar pattern as the official Docker images, e.g.

`bats:0.4.0-slim` being this minimal-style image
`bats:0.4.0` being a slightly bigger (batteries-included, probably Debian) image

As running "bats-in-container" has a few gotchas (as all containers do!) I'd be happy to write a guide. Have opened https://github.com/bats-core/bats-core/issues/30 as a placeholder.